v3/integrations/nrfiber/nrfiber.go

```diff
@@ -0,0 +1,216 @@
+// Copyright 2020 New Relic Corporation. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package nrfiber instruments https://github.com/gofiber/fiber applications.
+//
+// Use this package to instrument inbound requests handled by a fiber.App.
+// Call nrfiber.Middleware to get a fiber.Handler which can be added to your
+// application as a middleware:
+//
+//	app := fiber.New()
+//	// Add the nrfiber middleware before other middlewares or routes:
+//	app.Use(nrfiber.Middleware(newrelicApp))
+//
+// Example: https://github.com/newrelic/go-agent/tree/master/v3/integrations/nrfiber/example/main.go
+package nrfiber
+
+import (
+	"context"
+	"net/http"
+	"net/url"
+
+	"github.com/gofiber/fiber/v2"
+	"github.com/newrelic/go-agent/v3/internal"
+	"github.com/newrelic/go-agent/v3/newrelic"
+)
+
+func init() {
+	internal.TrackUsage("integration", "framework", "fiber", "v2")
```

Comment:
This should be "v1" as this tracks our integration version instead of the framework version